index_structure.h

```diff
@@ -37,6 +37,19 @@ class IndexStructure {
   // Returns true iff the stripe with `stripe_id` contains the given `value`.
   virtual bool StripeContains(size_t stripe_id, int value) const = 0;
 
+  // Returns a bitmap indicating possibly qualifying stripes for the given
+  // `value`. Probes up to `num_stripes` stripes.
```

Comment:
Perhaps add a note that classes extending IndexStructure can override this method when they can provide a optimized approach here.

Reply:
Done, thanks.